docs/simulate/quantum_virtual_machine.ipynb

```diff
@@ -252,9 +252,8 @@
     "For a hands-on example of the steps necessary to prepare a circuit to be run on the QVM, see the [QVM Basic Example](/cirq/simulate/qvm_basic_example) page.\n",
     "\n",
     "The steps necessary to make a circuit device-ready are summarized here:\n",
-    "1. Transform your circuit to use the correct gate set with `cirq.optimize_for_target_gateset`. Read [Transformers](/cirq/transform/transformers) for more on how to modify circuits. \n",
-    "2. Choose qubits on the virtual device for your circuit to run on. The connectivity required by your circuit must be supported by the connectivity present in your chosen qubit set. See [Qubit Picking](/cirq/hardware/qubit_picking) for more advice.\n",
-    "3. Map your transformed circuit to those qubits with `cirq.Circuit`'s `transform_qubits` function. This may require some careful planning depending on your particular circuit. \n",
+    "1. Map and route your circuit onto the device with `cirq.RouteCQC`. Read [Routing Transformer](/cirq/transform/routing_transformer) to learn how to fit a circuit onto a device. \n",
```

Comment:
One important note here is that `cirq.RouteCQC` expects your circuit to only contain 1 and 2 qubit gates. If the circuit contains multi-qubit gates; then those multi qubit gates must be "expanded" / "decomposed" in terms of 1/2q gates before calling the `cirq.RouteCQC` method. 

An example would be to add a `QuantumFourierTransformGate` on > 2 qubits. 

One easy way to do this is to run the `expand_composite` transformer before calling `cirq.RouteCQC`

```python
>>> cirq.expand_composite(circuit, no_decomp=lambda op: cirq.num_qubits(op) <= 2)
```

Expand composite is also run as part of `optimize_for_target_gateset`; so users can do:

1. `cirq.expand_composite` / `cirq.optimize_for_target_gateset`
2. `cirq.RouteCQC`
3. `cirq.optimize_for_target_gateset` (again).

Reply:
What parts of `optimize_for_target_gateset` does `RouteCQC` not respect that force us to run it a second time?

Reply:
No, because `RouteCQC` will insert `swap` gates, which might not be a native operation on all devices and therefore we'd need to decompose the newly inserted swaps into the target gateset that the device supports.